_data/team-members.yml

```diff
@@ -39,13 +39,13 @@ members:
     twitter: jvican
     photo: /resources/img/jorge.png
     currentlyWorkingOn: true
-    contact_for: [scalaPlatformProcess, sip, spores, buildTools]
+    contact_for: [scalaPlatformProcess, sip, buildTools, scalacProfiling]
```

Comment:
Can you add spores back? Sorry it was my mistake to ask you to remove it.

Reply:
Done!